lib/shimmer.dart

```diff
@@ -186,10 +185,10 @@ class _Shimmer extends SingleChildRenderObjectWidget {
   final Gradient gradient;
 
   const _Shimmer({
-    Widget child,
-    this.percent,
-    this.direction,
-    this.gradient,
+    Widget? child,
```

Comment:
Because the ```AnimatedBuilder```s  ```builder``` function returns a ```Widget?``` in ```_ShimmerState```, even if the child property is specified.
This could be fixed by using the ! operator. But the constructor of the ```SingleChildRenderObjectWidget```, that ```_Shimmer``` extends, defines its child as ```Widget?``` too.
So mainly for simplicity reasons and to avoid using the ! operator, because it doesn't change the meaning of the code.

Reply:
yep, it makes sense. Thanks for your contribution.